VideoLocker/res/layout/row_discussion_comments_comment.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.v7.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginTop="-5dp"
+    android:layout_marginLeft="@dimen/discussion_responses_standard_margin"
+    android:layout_marginRight="@dimen/discussion_responses_standard_margin"
+    app:cardCornerRadius="@dimen/edx_box_radius"
+    app:cardPreventCornerOverlap="false"
+    app:cardUseCompatPadding="true">
```

Comment:
Since this same `CardView` style is being used in a few places, you should probably create a global style for it.